src/pudl_archiver/archivers/nrelss.py

```
@@ -0,0 +1,194 @@
+"""Download NREL Standard Scenarios data."""
+
+import re
+
+from pudl_archiver.archivers.classes import (
+    AbstractDatasetArchiver,
+    ArchiveAwaitable,
+    ResourceInfo,
+)
+from pudl_archiver.utils import retry_async
+
+API_URL_PROJECTS_LIST = "https://scenarioviewer.nrel.gov/api/projects/"
+API_URL_FILE_LIST = "https://scenarioviewer.nrel.gov/api/file-list/"
+API_URL_FILE_DOWNLOAD = "https://scenarioviewer.nrel.gov/api/download/"
+
+REPORT_URL_PATTERN = re.compile(
+    r"https://www.nrel.gov/docs/(?P<fy>fy\d{2}osti)/(?P<number>\d{5}\.pdf)"
+)
+
+
+class AbstractNrelScenarioArchiver(AbstractDatasetArchiver):
+    """Base class for archiving projects from the NREL Scenario Viewer."""
+
+    project_year_pattern: re.Pattern
+    project_startswith: str
+    report_section: str
+    file_naming_order: tuple[str]
+
+    def handle_report_link_exceptions(self, project_year: str) -> str:
+        """Provide hard-coded exceptions for years where the normal report URL locator doesn't work."""
+        return ""
```

Non-blocking: Do you expect many years to not have a report? Right now this just silently ignores the problem which seems ok if this is uneven across years, but if this is undesired behavior we want to catch another option would be to return a `NotImplementedError`, which I think would more accurately reflect how you're trying to use this in the Abstract class. That way you would define the default cambium behavior for this in the cambium archiver file, rather than here.